### ISSUE TITLE
Variant error in turn based match

### ISSUE THREAD

**Issue by juanlu91** [playgameservices/play-games-plugin-for-unity#445]
I have 4 game modes in my app. When I try to start a new game, I set the variant to a value between 1 and 4, however, I see (logcat) that it is set to a big value (~4294967296)

I fixed it including this line in the "CreateWithInvitationScreen" method from NativeTurnBasedMultiplayerClient.cs

```
configBuilder.SetVariant(variant);
```

*CreateWithInvitationScreen:*

```
public void CreateWithInvitationScreen(uint minOpponents, uint maxOpponents, uint variant,
        Action<bool, TurnBasedMatch> callback) {
        callback = Callbacks.AsOnGameThreadCallback(callback);
        mTurnBasedManager.ShowPlayerSelectUI(minOpponents, maxOpponents, true, result => {
            if (result.Status() != GooglePlayGames.Native.Cwrapper.CommonErrorStatus.UIStatus.VALID) {
                callback(false, null);
            }

            using (var configBuilder = TurnBasedMatchConfigBuilder.Create()) {
                configBuilder.PopulateFromUIResponse(result);
				configBuilder.SetVariant(variant);
                using (var config = configBuilder.Build()) {
                    mTurnBasedManager.CreateMatch(config, BridgeMatchToUserCallback(callback));
                }
            }
        });
    }
```

Is it correct?

Thanks!

**Comment by claywilkinson** [comment]
Yes! that is correct.  Feel free to submit a pull request with the change - Otherwise, I'll add it to the list to update in the next release.

**Comment by juanlu91** [comment]
Ok, I'm a newbie but I'll try to submit the pull request anyway. 
Thanks so much for your answer, you are doing a great job!

**Comment by claywilkinson** [comment]
This has been implemented in the version 0.9.25 of the plugin
